PayButton/paybutton-server
# 5 [Issue #401] feat: delete wallet backend

**Description:**
Creates the backend for #401.

**Test plan:**
Should be covered by tests. Since it's only the backend and the endpoint requires session authentication, to actually test it you have to:
- open the devtools on your browser
- go to the Network tab,
- go to http://localhost:3000/wallet/2 for example, 
- see the 200 GET request, right click on it
- "Edit and resend"
- Change from GET to DELETE


**Depends on:**
- [x] #437
